book.toml

```diff
@@ -15,3 +15,5 @@ additional-js = [
 theme = "src/theme"
 
 [output.linkcheck]
+follow-web-links = true
+exclude = [ "www4\\.comp\\.polyu\\.edu\\.hk/~csxluo/" ]
```

Comment:
Why exclude this one? I'm fine with it, just think we should note the reason here so it's documented in the future.

Reply:
We have a link to a research paper hosted in this site. But their SSL cert is complained by link check. But accessing from the browser seems fine.